common/Resources/ti.internal/extensions/binding.js

```diff
@@ -83,3 +83,8 @@ export function redirectCoreModuleIdToPath(coreModuleId, internalPath) {
 export function bindObjectToCoreModuleId(coreModuleId, object) {
 	bindings.set(coreModuleId, object);
 }
+
+global.redirectCoreModuleIdToPath = redirectCoreModuleIdToPath;
+global.bindObjectToCoreModuleId = bindObjectToCoreModuleId;
+global.bindings = bindings;
+global.redirects = redirects;
```

Comment:
<!--
  1 failure:  common/Resources/...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: common/Resources/ti.internal/extensions/binding.js;
  Line: 90;
-->
- :no_entry_sign: common/Resources/ti.internal/extensions/binding.js line 90 – Newline required at end of file but not found. (eol-last)